web/controllers/github_api/http_client.ex

```diff
@@ -63,10 +63,25 @@ defmodule Dwylbot.GithubAPI.HTTPClient do
       %{comment: comment, url: url} ->
         url
         |> HTTPoison.post!(Poison.encode!(%{body: comment}), header(token))
+      %{add_assignees: assignees, url: url} ->
+        url
+        |> HTTPoison.post!(
+            Poison.encode!(%{assignees: assignees}),
+            header(token)
+          )
+      %{remove_assignees: assignees, url: url} ->
+        HTTPoison.request!(
```

Comment:
I'm using the ```request``` function instead of the ```delete``` as HTTPoison doesn't allow to pass a body with the this function